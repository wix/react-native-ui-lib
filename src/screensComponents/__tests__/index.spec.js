describe('screensComponents/index', () => {
  it('should export the same structure', () => {
    expect(require('../index')).toMatchSnapshot();
  });
});
