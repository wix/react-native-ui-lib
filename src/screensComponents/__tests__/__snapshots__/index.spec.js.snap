// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screensComponents/index should export the same structure 1`] = `
Object {
  "LoaderScreen": [Function],
  "Modal": [Function],
  "StateScreen": [Function],
}
`;
